solutions/cses-1093.mdx

```diff
@@ -127,4 +127,44 @@ public class TwoSets {
 }
 ```
 </JavaSection>
+
+<PySection>
+
+```py
+MOD = 10 ** 9 + 7
+
+
+def solve(n: int) -> int:
```

Comment:
why in function

Reply:
It's easy to break the input into two cases, odd and even, and return them at different points in the code. Although it's just as easy to remove the function I suppose. Do you want me to do that?

Reply:
yeah, do that & `sys.exit()`